readme.md

```diff
@@ -356,11 +356,12 @@ Check out my [blog](https://blog.sindresorhus.com) and follow me on [Twitter](ht
 ## Miscellaneous
 
 - [JSON](https://github.com/burningtree/awesome-json)
+- [GeoJSON](https://github.com/tmcw/awesome-geojson)
```

Comment:
I think it's best if you indent both of these and change `JSON Datasets` -> `Datasets`